rfcs/text/0004-session.md

```diff
@@ -88,62 +38,85 @@ Stage 3: Add or update all remaining field definitions. The list should now be e
 
 ## Usage
 
-Session fields are used to describe the sesison attributes of:
- - Client VPN Sessions
- - Network to Network VPN Sessions
- - Network Access Sessions (NAC, WPA, EAP, etc.)
- - Local or remote device login sessions (RDP, ICA, xWindows)
+Session fields are used to describe and track a discrete grouping of interactions, typically bounded by
+authentication or authorization events and tied to a specific user, application, or system component.
+
+For example:
+ - Network Access Sessions (NAC or Wireless LAN)
+ - Local or remote device login sessions (tty, console, ssh, RDP, ICA, xWindows, etc.)
+ - VPN Sessions (network to network, or client to network)
+ - Local or remote device login sessions (console, tty, RDP, ICA, xWindows, ssh, etc.)
  - Administrative sessions on infrastructure devices
  - Administrative sessions on cloud or application management portals
- - Applications sessions (e.g. sql server odbc session, application access session)
+ - Applications sessions (e.g. sql server odbc session, web cookie based session, application access session)
+ - Cloud API access sessions
 
 
 ## Source data
```

Comment:
For each of the log examples in the "Source data" section, could you include a breakdown of expected values for the session fields of each log event?

E.g. the ASA login could be something like

* session.kind: network
* session.authorization: admin
* session.type: [remote]

Not sure if one of the values in the ASA example is a session name or session id, but if so, list those as well.

Reply:
Yeah - probably wouldn't be a bad idea to fill in everything in some additional example.json file(s)?